website/docs/maintenance/observability/monitor-metrics.md

```diff
@@ -757,6 +757,16 @@ Some metrics might not be exposed when using other JVM implementations (e.g. IBM
       <td>endOffset</td>
       <td>The end offset in local storage for this table bucket.</td>
       <td>Gauge</td>
+    </tr>
+     <tr>
+      <td>lakeLagRecords</td>
```

Comment:
If modify accoding to my suggestion, then the `infix` should be `table_bucket_lakeTiering`